docs/intro.md

```diff
@@ -1,25 +1,46 @@
-Hackweek Guidebook
-==================
+Hackweeks As A Service
```

Comment:
i feel like a clearer title is needed, 'UW eScience Hackweek Services', 'Work with UW eScience to host a hackweek' ?

Reply:
agreed - "Partner with UW eScience to Host a Hackweek" came to mind before I realized it's almost verbatim what @scottyhq suggested!